datumaro/util/image.py

```diff
@@ -402,8 +402,8 @@ def load_image_meta_file(image_meta_path: str) -> ImageMeta:
     """
     Loads image metadata from a file with the following format:
 
-        <image name 1> <height 1> <width 1>
-        <image name 2> <height 2> <width 2>
+        <image name 1> <height 1> <width 1> |n
+        <image name 2> <height 2> <width 2> |n
```

Comment:
There is no point in using `|n` and `|s` - they are supposed for CLI output formatting in `CliPlugin`s.